templates/beta/contracts/MultisigTemplate.sol

```diff
@@ -4,25 +4,53 @@ import "./BetaTemplateBase.sol";
 
 
 contract MultisigTemplate is BetaTemplateBase {
-    function MultisigTemplate(DAOFactory _fac, MiniMeTokenFactory _minimeFac, APMRegistry _apm, EtherToken _etherToken, IFIFSResolvingRegistrar _aragonID, bytes32[4] _appIds)
-             BetaTemplateBase(_fac, _minimeFac, _apm, _etherToken, _aragonID, _appIds) public {}
-
-    function newToken(string name) external returns (MiniMeToken token) {
-        token = minimeFac.createCloneToken(address(0), 0, name, 0, "Multisig", true);
+    uint64 constant public VOTING_TIME = 157680000; // 5 years
+
+    function MultisigTemplate(
+        DAOFactory _fac,
+        MiniMeTokenFactory _minimeFac,
+        APMRegistry _apm,
+        EtherToken _etherToken,
+        IFIFSResolvingRegistrar _aragonID,
+        bytes32[4] _appIds
+    )
+        BetaTemplateBase(_fac, _minimeFac, _apm, _etherToken, _aragonID, _appIds) public
+    {}
+
+    function newToken(string name, string symbol) external returns (MiniMeToken token) {
+        token = minimeFac.createCloneToken(
+            address(0),
+            0,
+            name,
+            18,
+            symbol,
+            true
+        );
         cacheToken(token, msg.sender);
     }
 
     function newInstance(string name, address[] signers, uint256 neededSignatures) external {
         uint256[] memory stakes = new uint256[](signers.length);
 
         for (uint256 i = 0; i < signers.length; i++) {
-            stakes[i] = 1;
+            stakes[i] = 10**18;
         }
 
         MiniMeToken token = popTokenCache(msg.sender);
-        Voting voting = createDAO(name, token, signers, stakes, 1);
+        Voting voting = createDAO(
+            name,
+            token,
+            signers,
+            stakes,
+            10**18
```

Comment:
Mmm I think the intent was to create a non-transferrable "multisig" token here (so `maxToken <= 1`). That's why there was no decimals in the token and all the stakes were 1.

Was there a problem initializing it like it was before?

Reply:
Yes, the problem is because of this line:
https://github.com/aragon/aragon-apps/blob/master/templates/beta/contracts/MultisigTemplate.sol#L25
As we are dividing, we need decimals.

Reply:
Actually and sorry for being wrong the other day, I think the voting app should work even with simpler numbers because of how it does the calculations https://github.com/aragon/aragon-apps/blob/master/apps/voting/contracts/Voting.sol#L277

Reply:
Sorry for being wrong first ;-) And thanks @sohkai for pointing it out! Reverting those changes too.